src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,61 @@
 
 #include "t06_homework.h"
 #include <iostream>
-
+#include <string>
+#include <set>
 using namespace std;
-
 int t06_homework() {
+bool abc;
+	string word, task;
+	set <string> dict;
+	int n, k, x, osh = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'x' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4528206)